Renderer.js

```diff
@@ -0,0 +1,35 @@
+class View {
+    constructor() {
```

Comment:
why are you not using handlebars? You will need to use handlebars to pass the exam next week